resources/com.yubico.yubioath.appdata.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.yubico.yubioath</id>
+  <launchable type="desktop-id">com.yubico.yubioath.desktop</launchable>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD-2-Clause</project_license>
+
+  <name>Yubico Authenticator</name>
+  <summary>Graphical interface for displaying OATH codes with a YubiKey</summary>
+  <url type="homepage">https://developers.yubico.com/yubioath-desktop/</url>
+  <content_rating type="oars-1.1"/>
+
+  <developer_name>Yubico</developer_name>
+  <update_contact>andrey@sitnik.ru</update_contact>
+
+  <description>
+    <p>Generating Open Authentication (OATH) time-based TOTP and event-based HOTP one-time password codes, with the help of a YubiKey that protects the shared secrets.</p>
+    <p>Features:</p>
+    <ul>
+      <li>Add a credential by scanning a QR code on the screen</li>
+      <li>Favorite your most used credentials for easy access</li>
+      <li>Keep your secret seeds safe by storing them on a YubiKey</li>
+      <li>Require a touch on YubiKey to generate the code</li>
+      <li>Protect your credentials with a device password</li>
+      <li>Connect an external smart card reader and use the YubiKey over NFC</li>
+    </ul>
+  </description>
+
+  <screenshots>
+    ​<screenshot type="default">
+      <image type="source" width="850" height="393">https://raw.githubusercontent.com/Yubico/yubioath-desktop/master/screenshot.png</image>
+      <caption>Different modes and themes</caption>
+    </screenshot>
+  </screenshots>
+
+  <releases>
```

Comment:
I would drop the releases section for now and keep this minimal, since we already maintain the NEWS file.

Reply:
The releases are valuable info, it's used in Flathub/GNOME Software/KDE Discover/.. to show to the user which version is available/installed on their system. 

It's also a requirement for publishing an application on Flathub.

Reply:
The NEWS file, is mostly read by: contributors, collaborators, developers or someone involved with the code source. The releases tags here targets a general audience (less technical) I would say

Reply:
Didn't know that it was a requirement for Flathub. I'm happy to include these again as they were (sorry @ai ... ), and we'll look into later how to sync the content for them for upcoming releases.

Reply:
It shouldn't be hard with https://blog.tenstral.net/2020/01/a-big-appstream-status-update.html. There's now a `news-to-metainfo` command shipped with appstream-util if you want to automate the task.

Reply:
I reverted `releases` removal and add fresh 5.0.2 release there 5883176